packages/grid/x-grid-data-generator/LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2020 Material-UI SAS
+Copyright (c) 2021 Material-UI SAS
```

Comment:
AFAIK, this change is not correct, see https://opensource.stackexchange.com/questions/6389/how-do-the-years-specified-in-a-copyright-statement-work for why. It's meant to signal the first year when the content was created.

@mbrookes A preference on what we should do? Like React or

```suggestion
Copyright (c) Material-UI SAS
```

or as before, the first year

```suggestion
Copyright (c) 2020 Material-UI SAS
```

or a range

```suggestion
Copyright (c) 2020-2021 Material-UI SAS
```

?

I would be leaning to revert, match what we do in the core.